noggin/controller/password.py

```diff
@@ -126,7 +126,7 @@ def user_settings_password(ipa, username):
 def forgot_password_ask():
     form = ForgottenPasswordForm()
     if form.validate_on_submit():
-        username = form.username.data
+        username = form.username.data.lower()
```

Comment:
This will work, but I'd prefer adding a filter in the Form definition itself. This way if the form is ever used elsewhere, we won't have this bug again.
You can check how some form fields already use the `filters` argument for an example, and WTForms has documentation about it too.

Reply:
I added the new filter in `base.py` with the previous one, `strip`. Is it okay?

Reply:
Yes, that's perfect, thanks :-)